Assets/HoloToolkit/Input/Scripts/InputSources/IInputSource.cs

```diff
@@ -8,19 +8,6 @@
 namespace HoloToolkit.Unity.InputModule
 {
     /// <summary>
-    /// Flags used to indicate which input events are supported by an input source.
```

Comment:
Yeah, the diffs line up kind of funny here, but this whole set of flags was made obsolete in an earlier refactoring of InputManager that someone did.  We removed the flags, since they weren't used anymore.